src/configuration/customers/requisition-lists.md

```diff
@@ -13,3 +25,7 @@ Stores > Settings > [Configuration]({% link stores/configuration.md %}) > [Cust
 |Field|[Scope]({% link configuration/scope.md %})|Description|
 |--- |--- |--- |
 |Number of Requisition Lists|Store View|Determines the maximum number of requisition lists that can be created per customer account. The minimum number is 2, and the maximum number is 999.|
+
+The module does not create any backward incompatible changes. It can be deactivated and uninstalled in any time.
```

Comment:
Please remove this sentense.

THank you!

Reply:
done